backend/module/resolver/resolver.go

```diff
@@ -177,6 +186,28 @@ func (r *resolverAPI) GetObjectSchemas(ctx context.Context, req *resolverv1.GetO
 	}, nil
 }
 
+func (r *resolverAPI) Autocomplete(ctx context.Context, req *resolverv1.AutocompleteRequest) (*resolverv1.AutocompleteResponse, error) {
+	var err error
+	results := []*resolverv1.AutocompleteResponse_AutocompleteResult{}
+
+	// Iterate through all of the available resolvers & schemas to find the one requested
+	// If that schema exists then we call the associated autocomplete function for that resolver
+	for _, res := range resolver.Registry {
+		resSchema := res.Schemas()
+		if _, ok := resSchema[req.Want]; ok {
+			results, err = res.AutoComplete(ctx, req.Want, req.Search, req.ResultLimit)
+			if err != nil {
+				return nil, err
+			}
+			break
```

Comment:
i'm not sure if this is correct. we want to autocomplete from any resolver that can provide the object, right?

Reply:
Yeah good question, for the initial implementation I was going to leave this out until we decided on how we wanted to merge / sort results from different resolvers to surface the most relevant ones to the user. But maybe as a start it's fine if the list is not perfectly sorted and Im overthinking it. 

For now tho I can just merge these results together and will leave a comment as to what additional work needs to be done / thought about.

Reply:
For now you could just stop if limit satisfied? Later we can do Levenshtein distance or some more advanced algo.

Reply:
Alright yeah good suggestion let me start with that.